src/routes.js

```diff
@@ -36,6 +37,7 @@ export default function (props = {}) {
         <Route path='petitions/:id' component={Petition} />
         <Route path='petitions/:id/edit' component={EditPetition} />
         <Route path='petitions/:id/published' component={PublishedPetition} />
+        <Route path='respond/:token' component={RespondToPetition} />
```

Comment:
That was my first thought, but I noticed the convention `[Verb]:Petition` kinda emerged.